web/src/features/hfiCalculator/components/prepLevel.ts

```diff
@@ -1,10 +1,14 @@
 import { isUndefined } from 'lodash'
 
-export type PrepLevel = 1 | 2 | 3 | 4 | 5 | 6 | undefined
-export const calculatePrepLevel = (meanIntensityGroup: number | undefined): PrepLevel => {
+export const calculatePrepLevel = (
+  meanIntensityGroup: number | undefined
+): number | undefined => {
   // for now, prep level calculation assumed a fixed Fire Starts value of 0-1
+
   if (isUndefined(meanIntensityGroup)) {
     return undefined
+  } else {
+    meanIntensityGroup = Math.round(meanIntensityGroup)
```

Comment:
Isn't this a long way round to get it to floor?
e.g.
```
2.9
rounds to 3.
is smaller than 4.
gives 2.
```
easier to just say `Math.floor(meanIntensityGroup)`?

Reply:
Can't this entire bit of code just be replaced by a Math.floor instead of a bunch of if else?